bindings/py/tests/sdr/Metrics_test.py

```diff
@@ -86,6 +86,16 @@ def testAF_Example(self):
 """Activation Frequency Min/Mean/Std/Max 0.333333 / 0.5 / 0.166667 / 0.666667
 Entropy 0.918296""")
 
+    def testAF_initializeToValue(self):
+        X  = sdr.SDR( 10 )
+        AF = sdr.ActivationFrequency( X, period = 100)
+        AF.initializeToValue( .02 )
+        assert( np.all( AF.activationFrequency == .02 ))
+        X.randomize( 1 )
+        assert( np.all( AF.activationFrequency > .025 ))
```

Comment:
I will make this test deterministic so that it never fails CI.

I would like to keep the range checks because: the exact value depends on an arbitrary random seed which is not what this is testing.

Reply:
Actually, the seed is not needed since this uses 100% sparsity which is not very random.

The reason I use a range check here is because it true value is not 3%, it's very close to 3% but since it uses an exponential rolling avg it isn't quite 3%, it's a little lower.  I will change this to use exact equality instead of the current range check.